src/components/Table/__stories__/utils.tsx

```diff
@@ -54,6 +56,17 @@ export const columns: TableColumnConfig<DataItem>[] = [
     {
         id: 'name',
         name: 'Name',
+        template(item, index) {
+            if (index % 2 === 0) {
+                return item.name;
+            }
+            return (
+                <div>
+                    {item.name} <br />
+                    Lorem
```

Comment:
Better to split name by space instead of adding word

Reply:
[change](https://github.com/gravity-ui/uikit/pull/1149/files#diff-a2fac8f14d415f502e927594f887e36463ee190ef7b3a0bc0230b5a0bc19f0c4R63)